tools_cam/test_cam.py

```diff
@@ -71,7 +71,7 @@ def main():
     writer = SummaryWriter(log_dir)
 
     train_loader, val_loader = creat_data_loader(cfg, os.path.join(cfg.BASIC.ROOT_DIR, cfg.DATA.DATADIR))
-    device, model, optimizer, cls_criterion = creat_model(cfg, args)
```

Comment:
Remove optimizer because this function is not returning optimizer particularly defined in this file.